standalone/pruntime/enclave/src/contracts/mod.rs

```diff
@@ -95,6 +96,28 @@ pub mod serde_balance {
     }
 }
 
+/// Serde module to serialize or deserialize anyhow Errors
+pub mod serde_anyhow {
+    use crate::std::string::{String, ToString};
+    use anyhow::Error;
+    use serde::{Deserialize, Deserializer, Serialize, Serializer};
+
+    pub fn serialize<S>(value: &Error, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        let s = value.to_string();
+        String::serialize(&s, serializer)
+    }
+    pub fn deserialize<'de, D>(deserializer: D) -> Result<Error, D::Error>
+    where
+        D: Deserializer<'de>,
+    {
+        let s = String::deserialize(deserializer)?;
+        Ok(Error::msg(s))
+    }
+}
```

Comment:
@h4x3rotab This is the new change that I made in order to get it working for `Response`s that return an `anyhow::Error`.

Reply:
LGTM